tests/integration/kubernetes/gha-run.sh

```diff
@@ -207,7 +207,25 @@ function deploy_kata() {
 	[ "$(yq .image.tag ${values_yaml})" = "${DOCKER_TAG}" ] || die "Failed to set image tag"
 	echo "::endgroup::"
 
-	helm install kata-deploy "${helm_chart_dir}" --values "${values_yaml}" --namespace kube-system --debug
+	local max_tries=3
+	local interval=10
+	local i=0
+	# Retry loop for helm install to prevent transient failures due to instantly unreachable cluster
+	set +e # Disable immediate exit on failure
+	while true; do
+		helm install kata-deploy "${helm_chart_dir}" --values "${values_yaml}" --namespace kube-system --debug
```

Comment:
i am worried that helm will fail if rbac gets deployed, for instance ... in this case we better just wait more than retrigger the install.

Reply:
What about using `helm upgrade --install ` instead of `helm install` then?

Reply:
sounds better!

Reply:
Thanks for the comment. I missed thinking those remaining resources in the first try.